base_menu_visibility_restriction/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+To use this module, you need to go to:
+
+* Activate the developer mode
+* Settings > Users & Companies > Groups and create new record(s) with "Name" of your choice and add to "Users" the ones you want to be able to access a certain Menu Item
+* Settings > Technical > User interface > Menu Items, search "Apps" record and update "Excluded groups"  with the one(s) you created earlier.
+* Login as demo user and check that "Apps" menu item not show.
```

Comment:
Several things:

* You have to put generic instructions, not a specific case. You can afterwards specify an example.
* Each step is a point. Don't group several of them together.
* Use numbered bullets.
* The group creation is out of the scope of this module.
```suggestion
#. Go to *Settings > Technical > User interface > Menu Items*.
#. Search for any menu and edit it.
#. Update "Excluded groups" with one group.
#. Login with a user of that group, and you won't see such menu.

You can try with demo data for the menu X and user demo.
```
And add such demo data :wink: